### PR TITLE
Drop Settlement, Caution Flag Rule, Rule, Sec702 School, School Closure tables

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/11354
Dropping unused tables, usage of tables was removed in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/694
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: db/migrate/20200814183045_drop_settlement.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class DropSettlement < ActiveRecord::Migration[5.2]
+  def change
+    if table_exists?("settlements")
+      drop_table :settlements
+    end
```
Comment (f1337) [inline db/migrate/20200814183045_drop_settlement.rb:5]:
```suggestion
    drop_table :settlements, if_exists: true
```

File: db/migrate/20200814183406_drop_sec702_school.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class DropSec702School < ActiveRecord::Migration[5.2]
+  def change
+    if table_exists?("sec702_schools")
+      drop_table :sec702_schools
+    end
```
Comment (f1337) [inline db/migrate/20200814183406_drop_sec702_school.rb:5]:
```suggestion
    drop_table :sec702_schools, if_exists: true
```

File: db/migrate/20200814183445_drop_school_closure.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class DropSchoolClosure < ActiveRecord::Migration[5.2]
+  def change
+    if table_exists?("school_closures")
+      drop_table :school_closures
+    end
```
Comment (f1337) [inline db/migrate/20200814183445_drop_school_closure.rb:5]:
```suggestion
    drop_table :school_closures, if_exists: true
```

File: db/migrate/20200814183702_drop_caution_flag_rules.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class DropCautionFlagRules < ActiveRecord::Migration[5.2]
+  def change
+    if table_exists?("caution_flag_rules")
+      drop_table :caution_flag_rules
+    end
```
Comment (f1337) [inline db/migrate/20200814183702_drop_caution_flag_rules.rb:5]:
```suggestion
    drop_table :caution_flag_rules, if_exists: true
```

File: db/migrate/20200814183825_drop_rule.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class DropRule < ActiveRecord::Migration[5.2]
+  def change
+    if !table_exists?("caution_flag_rules") && table_exists?("rules")
+      drop_table :rules
+    end
```
Comment (f1337) [inline db/migrate/20200814183825_drop_rule.rb:5]:
```suggestion
    drop_table :rules, if_exists: true
```

**Review by f1337 — APPROVED** [review]
Looking good! I made some non-blocking suggestions to make the migrations more Rails-y.
Muppet with sunglasses saying 'Looking good'

**Pull request merged**